raw-svo.dict.lua

```diff
@@ -15183,6 +15183,104 @@ affinity = {
       end
     }
   },
+  lyre = {
+    physical = {
+      aspriority = 0,
+      spriority = 0,
+      balanceful_act = true,
+      def = true,
+      undeffable = true,
+
+      isadvisable = function ()
+        return (not defc.lyre and not doingaction("lyre") and ((sys.deffing and defdefup[defs.mode].lyre) or (conf.keepup and defkeepup[defs.mode].lyre)) and not will_take_balance() and not conf.lyre_step and not affs.prone and (defc.dragonform or (conf.lyrecmd and conf.lyrecmd ~= "intone kail") or bals.word)) or false
+      end,
+
+      oncompleted = function ()
+        defences.got("lyre")
+
+        if conf.lyre then conf.paused = true; raiseEvent("svo config changed", "paused") end
+      end,
+
+      ontimeout = function()
+        if conf.paused and not defc.lyre then
+          echof("Lyre strum didn't happen - unpausing.")
```

Comment:
That'd be inconsistent with grove barrier and nightingale melody. Both talk about lyre, but are not 'lyres'. Also as soon as the depthswalker is dragon or uses a lyre as depthswalker by configuring the command, the echo doesn't fit either.

An additional point is that the aliases are "lyr", "lyc" and "lyf", which suggests lyre as well.